assets/dev/js/editor/editor-base.js

```diff
@@ -1255,9 +1255,13 @@ export default class EditorBase extends Marionette.Application {
 			devices = this.breakpoints.getActiveBreakpointsList( { largeToSmall: true, withDesktop: true } ),
 			newControlsStack = {};
 
+		// Move desktop to be the fist device, so desktop will the the parent of all devices.
```

Comment:
Instead of that, omit `withDesktop` parameter in the call above, then just unshift `desktop` to the `devices` array